src/amo/pages/AddonVersions/index.js

```diff
@@ -165,6 +166,21 @@ export class AddonVersionsBase extends React.Component<InternalProps> {
             header={header || <LoadingText />}
           >
             <ul>
+              <li className="AddonVersions-warning">
+                <Notice type="warning">
+                  <div className="AddonVersions-warning-text">
```

Comment:
Shouldn't we use a `p` instead of a `div`?

Reply:
I started with a `p` but it did bad things with the formatting. I guess I can undo all of those with additional css, so I will give that a try.